packages/moonstone/styles/colors.less

```diff
@@ -145,6 +145,7 @@
 @moon-header-sub-title-color:         #a6a6a6;
 @moon-sub-header-text-color:          @moon-header-sub-title-color;
 @moon-sup-header-text-color:          @moon-text-color;
```

Comment:
This part of this file is heavily crufted. "sup" header (short for superscript) was removed ages ago, but parts still remain simply because we've not gone in to remove the old bits (low value), but in the interest of tightening this code up and having only meaningful code bearing on our maintenance cost, I think it's valid to remove these abandoned variables.